http/src/test/java/io/hyperfoil/http/TwoServersTest.java

```diff
@@ -37,15 +40,16 @@ protected void initRouter() {
       });
    }
 
-   @Override
-   public void before(TestContext ctx) {
-      super.before(ctx);
+   @BeforeEach
+   public void before(Vertx vertx, VertxTestContext ctx) {
+      super.before(vertx, ctx);
       Router secondRouter = Router.router(vertx);
       secondRouter.route("/test").handler(context -> context.response().setStatusCode(300).end());
       secondServer = vertx.createHttpServer().requestHandler(secondRouter)
-            .listen(0, "localhost", ctx.asyncAssertSuccess(srv -> {
+            .listen(0, "localhost", ctx.succeeding(srv -> {
                benchmarkBuilder.plugin(HttpPluginBuilder.class)
                      .http("http://localhost:" + secondServer.actualPort()).endHttp();
+               ctx.completeNow();
             }));
```

Comment:
> Interesting, after rebasing a test started failing in the CI:
> 
> ```
> TwoServersTest.testTwoServers:76->BaseScenarioTest.runScenario:25 » BenchmarkDefinition Phase test, sequence test, step httpRequest (1/3): GET to localhost:0/test is invalid - no HTTP configuration defined.
> ```
> 
> Need to investigate it a bit more!

The issue seems here, the HTTP plugin is added only once the server is up and running but the test itself does not actually wait for it.. so it might happen that the test starts before the server start listening, thus the issue where Hyperfoil cannot find the http plugin defined in the benchmark.

Was able to simulate locally by adding a `Thread.sleep` just before `benchmarkBuilder.plugin(HttpPluginBuilder.class)`.